fiftyone/utils/labels.py

```
@@ -1198,3 +1205,208 @@ def _get_filepath(sample_or_frame, sample):
+                context.save(sample)
+
+            count += 1
+            if count % batch_size == 0:
+                dataset.set_values(out_field, values_2d_label, key_field="id")
+                values_2d_label.clear()
+
+    if values_2d_label:
+        dataset.set_values(out_field, values_2d_label, key_field="id")
+
+
+def _process_3d_sample(
```

Pylint (3.3.7)

[refactor] Too many positional arguments (8/5)

(R0917)